src/main/resources/AWS_Instance_Details.csv

```diff
@@ -1,118 +1,184 @@
-C5 High-CPU Extra Large,c5d.xlarge,8.00,4,0.192,0.192,0.69,0,None,0,Intel Xeon Platinum 8124M,3,100 GiB NVMe SSD,Up to 10 Gigabit,3500,437.5,TRUE,TRUE,0.121,0
-G4DN 12xlarge,g4dn.12xlarge,192.00,48,3.912,3.912,8.56,4,NVIDIA T4 Tensor Core,64,Intel Xeon Family,0,900 GiB NVMe SSD,50 Gigabit,7000,875,TRUE,FALSE,2.465,0
-G4DN 16xlarge,g4dn.16xlarge,256.00,64,4.352,4.352,11.41,1,NVIDIA T4 Tensor Core,16,Intel Xeon Family,0,900 GiB NVMe SSD,50 Gigabit,7000,875,TRUE,FALSE,2.742,0
-G4DN Double Extra Large,g4dn.2xlarge,32.00,8,0.752,0.752,1.43,1,NVIDIA T4 Tensor Core,16,Intel Xeon Family,0,225 GiB NVMe SSD,Up to 25 Gigabit,3500,437.5,TRUE,FALSE,0.474,0
-G4DN Quadruple Extra Large,g4dn.4xlarge,64.00,16,1.204,1.204,2.85,1,NVIDIA T4 Tensor Core,16,Intel Xeon Family,0,225 GiB NVMe SSD,Up to 25 Gigabit,3500,437.5,TRUE,FALSE,0.759,0
-G4DN Eight Extra Large,g4dn.8xlarge,128.00,32,2.176,2.176,5.7,1,NVIDIA T4 Tensor Core,16,Intel Xeon Family,0,900 GiB NVMe SSD,50 Gigabit,7000,875,TRUE,FALSE,1.371,0
-G4DN Extra Large,g4dn.xlarge,16.00,4,0.526,0.526,0.71,1,NVIDIA T4 Tensor Core,16,Intel Xeon Family,0,125 GiB NVMe SSD,Up to 25 Gigabit,3500,437.5,TRUE,FALSE,0.331,0
-I2 Double Extra Large,i2.2xlarge,61.00,8,1.705,1.705,3,0,None,0,Intel Xeon E5-2670 v2 (Ivy Bridge),2.5,1600 GiB (2 * 800 GiB SSD),High,1000,125,FALSE,TRUE,0.848,0.05
-I2 Quadruple Extra Large,i2.4xlarge,122.00,16,3.41,3.41,6,0,None,0,Intel Xeon E5-2670 v2 (Ivy Bridge),2.5,3200 GiB (4 * 800 GiB SSD),High,2000,250,FALSE,TRUE,1.696,0.1
-I2 Eight Extra Large,i2.8xlarge,244.00,32,6.82,6.82,12,0,None,0,Intel Xeon E5-2670 v2 (Ivy Bridge),2.5,6400 GiB (8 * 800 GiB SSD),10 Gigabit,0,0,FALSE,TRUE,3.392,0
-I2 Extra Large,i2.xlarge,30.50,4,0.853,0.853,1.5,0,None,0,Intel Xeon E5-2670 v2 (Ivy Bridge),2.5,800 GiB SSD,Moderate,500,62.5,FALSE,TRUE,0.424,0.02
-I3 High I/O 16xlarge,i3.16xlarge,488.00,64,4.992,4.992,16,0,None,0,Intel Xeon E5-2686 v4 (Broadwell),2.3,15200 GiB (8 * 1900 GiB NVMe SSD),20 Gigabit,14000,1750,FALSE,TRUE,3.427,0
-I3 High I/O Double Extra Large,i3.2xlarge,61.00,8,0.624,0.624,2,0,None,0,Intel Xeon E5-2686 v4 (Broadwell),2.3,1900 GiB NVMe SSD,Up to 10 Gigabit,1700,212.5,FALSE,TRUE,0.428,0
-I3 High I/O Quadruple Extra Large,i3.4xlarge,122.00,16,1.248,1.248,4,0,None,0,Intel Xeon E5-2686 v4 (Broadwell),2.3,3800 GiB (2 * 1900 GiB NVMe SSD),Up to 10 Gigabit,3500,437.5,FALSE,TRUE,0.857,0
-I3 High I/O Eight Extra Large,i3.8xlarge,244.00,32,2.496,2.496,8,0,None,0,Intel Xeon E5-2686 v4 (Broadwell),2.3,7600 GiB (4 * 1900 GiB NVMe SSD),10 Gigabit,7000,875,FALSE,TRUE,1.714,0
+Name,API_Name,Memory_GB,vCPUs,Compute_Contract_Price,On_Demand_Cost_Hourly,Hourly_DBUs,GPUs,GPU_Model,GPU_Memory_GB,Physical_Processor,Clock_Speed_GHz,Instance_Storage,Network_Performance,EBS_Optimized_Max_Bandwidth_Mbps,EBS_Optimized_Max_Throughput_128K_Mbps,EBS_As_NVMe,Enhanced_Networking,Linux_Reserved_Cost_Hourly,EBS_Optimized_Surcharge_Hourly

```

Comment:
The .00 was removed from the Memory_GB field. This is not good because if schema is inferred it gets inferred as integer and not double. It must be inferred as double to ensure this works for all node types across all clouds.